changelog/6404.feature.md

```diff
@@ -0,0 +1 @@
+User can use ``rasa data convert {nlu|core} -f yaml`` command to convert training data from Markdown format to YAML format.
```

Comment:
what do I need to do to make this support nlg (responses) as well?

Reply:
`NLGMarkdownReader::reads` returns `TrainingData`as well as `MarkdownReader`, so it's mostly about:
- writing tests for `RasaYamlWriter` making sure the conversion is correct
- extending the "convert_to_yaml" function from this review to support `NLGMarkdownReader`